source/reference/minio-mc/mc-version.rst

```diff
@@ -60,6 +60,11 @@ Parameters
    Mutually exclusive with :mc-cmd:`~mc version suspend` and
    :mc-cmd:`~mc version info`
 
+   .. mc-cmd:: enable --excluded-prefixes
```

Comment:
```suggestion
.. mc-cmd:: --excluded-prefixes
```

Since this is nested, it inherits the parent - going all the way up the chain if necessary